extensions/blocks/podcast-player/components/header.js

```diff
@@ -3,12 +3,67 @@
  */
 import { memo } from '@wordpress/element';
 
-const Header = memo( ( { track, children } ) => (
-	<div>
-		{ /* TODO: display info based on `track` */ }
-		{ track ? <div>{ track.title }</div> : null }
-		<div>{ children }</div>
+const Header = memo( ( { playerId, title, cover, link, track, children } ) => (
+	<div className="jetpack-podcast-player__header-wrapper">
+		<div className="jetpack-podcast-player__header" aria-live="polite">
+			{ cover ? (
+				<div className="jetpack-podcast-player__track-image-wrapper">
+					{ /* alt="" will prevent the src from being announced. Ideally we'd have a cover.alt, but we can't get that from the RSS */ }
+					<img className="jetpack-podcast-player__track-image" src={ cover } alt="" />
+				</div>
+			) : null }
+
+			{ title || ( track && track.title ) ? (
+				<div className="jetpack-podcast-player__titles">
+					<Title playerId={ playerId } title={ title } link={ link } track={ track } />
+				</div>
+			) : null }
+		</div>
+
+		{ /* putting this above the audio player for source order HTML with screen readers, then visually switching it with the audio player via flex */ }
+		{ track && track.description ? (
+			<div
+				id={ `${ playerId }__track-description` }
+				className="jetpack-podcast-player__track-description"
+			>
+				{ track.description }
+			</div>
+		) : null }
+
+		{ /* children contains the audio player */ }
+		{ children }
 	</div>
 ) );
 
+const Title = ( { playerId, title, link, track } ) => {
+	return (
```

Comment:
Is there a reason we have a statement in `Title` and `PodcastTitle` instead of an expression like in `Header`?